network-primitives/bootstrap_node_provisioner.tf

```diff
@@ -156,7 +174,7 @@ resource "null_resource" "start-boostrap-nodes" {
       "systemctl reenable subspace.service",
 
       # set hostname
-      "hostnamectl set-hostname ${var.network-name}-bootstrap-node-${count.index}",
```

Comment:
why change the hostname for bootstrap node with domain prefix ?

Reply:
I am trying to make the hostnames as close as possible to dns and also machine names in provider dashboard.

Reply:
Sure but this not represent any closer TBH. Domain prefix is used for RPC 
Are you trying to set the where node is located on hetzner?